content/kb/deployments/invoking-python-subprocess-deployed-streamlit-app.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Invoking a Python subprocess in a deployed Streamlit app
+slug: /knowledge-base/deploy/invoking-python-subprocess-deployed-streamlit-app
+---
+
+# Invoking a Python subprocess in a deployed Streamlit app
+
+## Problem
+
+Let's suppose you want to invoke a subprocess to run a Python script `script.py` in your deployed Streamlit app `streamlit_app.py`.
```

Comment:
As we discussed, maybe add another sentence like:

"For example, the popular machine learning library Ludwig is run using a command-line interface, or maybe you want to run a bash script or similar type of process..."

Reply:
Added it in, take a look @snehankekre just to make sure it seems reasonable, then this is ready for merge.

Reply:
Thanks for making the change, @randyzwitch. LGTM!